eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/ConfigResolver/SiteAccessGroupConfigResolver.php

```diff
@@ -48,4 +111,19 @@ protected function getParameterFromResolver(SiteAccess $siteAccess, string $para
 
         throw new ParameterNotFoundException($paramName, $namespace, $triedScopes);
     }
+
+    protected function getParameterFromResolverForGroup(SiteAccessGroup $siteAccessGroup, string $paramName, string $namespace)
+    {
+        $groupScopeParamName = $this->resolveScopeRelativeParamName($paramName, $namespace, $siteAccessGroup->getName());
+        if ($this->container->hasParameter($groupScopeParamName)) {
+            return $this->container->getParameter($groupScopeParamName);
```

Comment:
To my taste it would be better to invert `if` statement so that `return` could be the last operation in method's body. Also empty line between variable assignment and `if` block for clarity.